doc/base/classes.xml

```diff
@@ -49422,7 +49429,7 @@
 		 - Rounded corners (individual radius for each corner)
 		 - Shadow
 		 About corner radius:
-		 	Setting corner radius to high values is allowed. As soon as corners would overlap the stylebox will switch to a relative system. Example: 
+		 	Setting corner radius to high values is allowed. As soon as corners would overlap the stylebox will switch to a relative system. Example:
```

Comment:
Avoid passive voice:
```Setting corner radius to high values is allowed.```
-> direct voice is always shorter, and a lot easier to understand for non native and young speakers
```You may use high values for 'corner radius'.```